codegenerator/cli/src/config_parsing/system_config.rs

```diff
@@ -853,54 +853,24 @@ impl SystemConfig {
 
 type ServerUrl = String;
 
-#[derive(Debug, Serialize, Clone, PartialEq)]
-pub struct HypersyncConfig {
-    pub endpoint_url: ServerUrl,
-    pub is_client_decoder: bool,
-}
-
-#[derive(Debug, Serialize, Clone, PartialEq)]
-pub struct HyperfuelConfig {
-    pub endpoint_url: ServerUrl,
-}
-
-#[derive(Debug, Serialize, PartialEq, Clone)]
-pub struct SyncConfig {
-    pub initial_block_interval: u32,
-    pub backoff_multiplicative: f64,
-    pub acceleration_additive: u32,
-    pub interval_ceiling: u32,
-    pub backoff_millis: u32,
-    pub query_timeout_millis: u32,
-    pub fallback_stall_timeout: u32,
-}
-
-impl Default for SyncConfig {
-    fn default() -> Self {
-        const QUERY_TIMEOUT_MILLIS: u32 = 20_000;
-        Self {
-            initial_block_interval: 10_000,
-            backoff_multiplicative: 0.8,
-            acceleration_additive: 500,
-            interval_ceiling: 10_000,
-            backoff_millis: 5000,
-            query_timeout_millis: QUERY_TIMEOUT_MILLIS,
-            fallback_stall_timeout: QUERY_TIMEOUT_MILLIS / 2,
-        }
-    }
-}
-
-#[derive(Debug, Serialize, PartialEq, Clone)]
-pub struct RpcConfig {
-    pub urls: Vec<String>,
-    pub sync_config: SyncConfig,
+// This data structure mainly needed to conviniently prepare data
+// for ConfigYAML, so we don't break backward compatibility
```

Comment:
It can just be nice for the editor docs and rust doc.
```suggestion
/// This data structure mainly needed to conviniently prepare data
/// for ConfigYAML, so we don't break backward compatibility
```

Reply:
Done in the https://github.com/enviodev/hyperindex/pull/467